src/platform/site-wide/sass/right-rail.scss

```diff
@@ -16,13 +16,14 @@
     margin-bottom: 3em !important;
   }
 
-  &:first-child {
-    background-color: $color-primary-lightest;
-    padding: 1em;
+  &.hub-promo {
+    background-color: $mm-gray;
     border: none;
 
     img {
-      margin-bottom: 1em;
+    }
+    section {
+      padding: 1em;
```

Comment:
Instead of using `section`, can we add a CSS class to [that element](https://github.com/department-of-veterans-affairs/vets-website/pull/9286/files#diff-fc57d82cdfb051d0f74e04afd93e82f6R24) and use that to stylize it? Something like `hub-promo-text` or `hub-promo-content` might be more descriptive than just `section`. 

Since there isn't any style, we can also just remove the `img` selector, although sometimes it's helpful to leave it in case we do end up adding style to it later.

Reply:
### @ncksllvn I added hub-promo-text classname to <section> and removed img selector.